### ISSUE TITLE
NMS missing in inference

### ISSUE THREAD

**Issue by josemafuen** [naseemap47/YOLO-NAS#52]
The training/testing allows it, but the inference script has no support for NMS, which makes me think it is not carried out. It would be nice to add it, in order to be able to test the models more efficiently.

**Comment by naseemap47** [comment]
Hi @josemafuen,
The training setup I used from [super-gradients](https://github.com/Deci-AI/super-gradients)
We have to give suggestion for them to add this feature.
